### PR TITLE
Feature/add exception handling

### PR DESCRIPTION
**Pull request by mSunberg**
## Description
Added new middleware "SlackExceptionNotification" for catching unknown exceptions and sending notification to #test-varslinger on slack.

## Related Issue(s)
- https://github.com/Altinn/altinn-correspondence/issues/389

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- [x] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [x] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

## Summary by CodeRabbit

- **New Features**
	- Introduced Slack integration for notifications, replacing email notifications.
	- Added a new environment variable for Slack URL in the container app configuration.

- **Bug Fixes**
	- Removed outdated alerting resources to streamline monitoring.

- **Documentation**
	- Updated workflows to include new Slack URL input for deployment notifications.

- **Chores**
	- Added new configuration settings for Slack URL in the development environment.

- **Refactor**
	- Enhanced error handling with centralized Slack exception notifications.

### CODE REVIEW

File: src/Altinn.Broker.Integrations/Slack/SlackDevClient.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
+using Slack.Webhooks;
+
+namespace Altinn.Correspondence.Integrations.Slack
+{
```
[the remark this answers is not in the document]
Comment (mSunberg) [reply src/Altinn.Broker.Integrations/Slack/SlackDevClient.cs:4]: Done.

File: src/Altinn.Broker.API/appsettings.Development.json [hunk 2]
```diff
@@ -33,5 +33,9 @@
     "Scope": "altinn:events.publish altinn:events.publish.admin altinn:register/partylookup.admin altinn:authorization/authorize.admin",
     "EncodedJwk": "",
     "ExhangeToAltinnToken": true
+  },
+  "GeneralSettings": {
+    "SlackUrl": "",
+    "CorrespondenceBaseUrl": "https://localhost:7241/"
```
[the remark this answers is not in the document]
Comment (mSunberg) [reply src/Altinn.Broker.API/appsettings.Development.json:39]: Skal jeg fjerne dette?

[the remark this answers is not in the document]
Comment (Andreass2) [reply src/Altinn.Broker.API/appsettings.Development.json:39]: Trenger ikke fjerne det, vi bruker nå appsettings.local.json

**Review by Andreass2 — APPROVED** [review]
Looks good. 

Remember to go through CodeRabbit comments before sending out the PR

**Review by Andreass2 — DISMISSED** [review]
Add labels

**Review by Andreass2 — COMMENTED** [review]
Check why the test crashes and create a issue if unrelated bug

**Review by Andreass2 — APPROVED** [review]

File: .github/actions/update-infrastructure/action.yml [hunk 3]
```diff
@@ -39,6 +39,9 @@ inputs:
   PLATFORM_SUBSCRIPTION_KEY:
     description: "Subscription key for platform"
     required: true
+  SLACK_URL:
+    description: "Email for Slack channel to post to"
```
Comment (CelineTrammi) [inline .github/actions/update-infrastructure/action.yml:43]: Typo 😅

**Review by CelineTrammi — APPROVED** [review]
Great job! :D

**Pull request merged**